audio-recorder.js

```diff
@@ -177,12 +165,9 @@ module.exports = class AudioRecorder {
       throw new Error('No live audio input available or permitted');
```

Comment:
This is now a promise catch callback, so return Promise.reject instead of throwing an error.